# MutaScript

## Introduction  

Up to a few years, linkage analysis using genetic markers or Sanger sequencing were massively used to identify genetic disorder in clinical research. These technic were extremely time and money expensive and in 2011 the genetic cause of over 3,500 Mendelian disorders remained unknow [1]. The advent of New Generation Sequencing (NGS) has immediately initiate the new ere of clinical research in genomic by bringing the sequencing of entire genomes (WGS) or exome (WES) economically feasible for many small laboratories. This technological leap has permit a great advance in the field of clinical research linking more than … genetic cause to a mendelian disorder [need citation here].
However, despite all of that, NGS bring with him new problematics. Indeed, with over 30,000 variants per individual exome and surrounding 10,000 of which are predicted to lead to a nonsynonymous substitution, a modification of a splice sites, or to a small insertion / deletion (indel) [need citation], finding the disease-causing variant became to be the daily labor-intensive task of geneticists. Even more that this new task regrouping among other things informatics and statistic is far from traditional geneticist skills. 
Usually NGS data analysis is divided in three principal steps. The first one is the raw data pre-processing which mainly consists to mapping raw data to a reference genome (may be cite some soft which do that?). Secondly, the analysis of mismatches between sample data and the reference sequence draw up a list of Single Nucleotide Variation (SNV) and small insertion / deletion (Indels). As said previously this list can enumerate more than 30,000 variation per individual (may be cite some soft which do that?). Finally, the analyze of these variants, including the annotation and filtering, is often weakness of genotype-phenotype analysis. Indeed, in most of case it is not enough to obtain a small human interpretable list and the causal variant is drowned in a multitude of non-causal variant. 
To improve the quality of variant filtering and so the efficiency of phenotype-genotype analysis we develop MutaScript. This score rank transcript considering their probability of being link to a mendelian disorder. For that we assumed that most mutated transcripts in general population were probably not involved in any severe Mendelian disease. MutaScript score of a transcript combines the allelic frequency of each variant displayed on ExAC [2] database overlapping this transcript as well as their impact predicted by Variant Effect Predictor (VEP) [36]. So, higher is the variant impact, higher will be its MutaScript score contribution. 
Ranking transcript (or gene) considering their variation load was already done by the Residual Variation Intolerance Score (RVIS) [4] and the Probability of loss-of-function (LoF) Incoherency (pLI) [2] scores. However, MutaScript differs from them on several points. Firstly, like pLI, MutaScript give a score to each protein-coding transcript where RVIS only scored genes which leads to a loss of information from RVIS. Moreover, RVIS and pLI only consider LoF variants, mainly splice donor / acceptor variants, nonsense or frameshift variants which represent only …% of ExAC variants. MutaScript consider all variants whatever its consequence, and it weight its score contribution considering the predicted deleteriousness of its impact. Also, we noticed a strong correlation between the RVIS score of a gene and the pLI score of a transcript and the CDS size of this gene (or transcript). Because this correlation is mostly due to a bias of their score formula and not because a biological fact, we build MutaScript to avoid this correlation which can lead to interpretation errors. To validate our scores we used the Human Phenotype Ontology (HPO) [5] and demonstrate that MutaScript is more…

## Matériel & Méthodes  

### Récupération des données ExAC, filtrage et pré analyse

## Résultats  

### Définition de la formule de score  

### Analyse de la corrélation  

### Analyse HPO  

## Conclusion  




<!-- 1.	Stitziel, N. O., Kiezun, A. & Sunyaev, S. Computational and statistical approaches to analyzing variants identified by exome sequencing. Genome Biol. 12, 227 (2011).-->
<!-- 2.	Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016). -->
<!-- 3.	McLaren, W. et al. The Ensembl Variant Effect Predictor. Genome Biol. 17, 122 (2016). -->
<!-- 4.	Petrovski, S. et al. Genic Intolerance to Functional Variation and the Interpretation of Personal Genomes. PLoS Genet. 9, e1003709 (2013). -->
<!-- 5.	Köhler, S. et al. The Human Phenotype Ontology in 2017. Nucleic Acids Res. (2016). doi:10.1093/nar/gkw1039 -->



