`r if(knitr:::is_latex_output()) '\\appendix'`

`r if(!knitr:::is_latex_output()) '# (APPENDIX) Appendix {-}'` 

# Article annexe 1 {#dnah12014}  

**Mutations in DNAH1, which Encodes an Inner Arm Heavy Chain Dynein, Lead to Male Infertility from Multiple Morphological Abnormalities of the Sperm Flagella**

Ben Khelifa M^\*^, Coutton C^\*^, Zouari Raoudha, **Karaouzène T**, Rendu J, Bidart M, Yassine S, Pierre V, Delaroche J, Hennebicq S, Grunwald D, Escalier D, Pernet-Gallay K, Jouk PS, Thierry-Mieg N, Touré A, Arnoult C, Ray PF

American Journal of Human Genetics, Janvier 2014  

^\*^ Co-premiers auteurs

\newpage

\includepdf[pages=-]{bib/DNAH1_2014.pdf}

\newpage

# Tables des variants restant après application des filtres pour les cas familiaux 

```{r tabmmaf1, cache=TRUE, dependson="dfrecapmmaf"}

caption <- "Liste des variants ayant passé l'ensemble des filtres pour les deux fères P1 et P2 de la famille MMAF1"

df_filter_fam %>% ungroup %>% mutate(`HGVSc, HGVSp` = paste0(na.replace(HGVSc, ".")," ; ", na.replace(HGVSp, "."))) %>% filter(FAM == "MMAF1", CANONICAL == "YES") %>% 
  select(SYMBOL, `HGVSc, HGVSp`, Consequence) %>%  
  distinct() %>%
  kable(format = "latex",
        booktabs = TRUE,
        longtable = TRUE,
        caption = caption) %>% 
  add_header_above(c(" ", "Variant impact" = 2))

```

```{r tabmmaf2, cache=TRUE, dependson="dfrecapmmaf"}

caption <- "Liste des variants ayant passé l'ensemble des filtres pour les deux fères P3 et P4 de la famille MMAF2"

df_filter_fam %>% ungroup %>% mutate(`HGVSc, HGVSp` = paste0(na.replace(HGVSc, ".")," ; ", na.replace(HGVSp, "."))) %>% filter(FAM == "MMAF2", CANONICAL == "YES") %>% 
  select(SYMBOL, `HGVSc, HGVSp`, Consequence) %>%  
  distinct() %>%
  kable(format = "latex",
        booktabs = TRUE,
        longtable = TRUE,
        caption = caption) %>% 
  add_header_above(c(" ", "Variant impact" = 2))

```

```{r tabmmaf3, cache=TRUE, dependson="dfrecapmmaf"}

caption <- "Liste des variants ayant passé l'ensemble des filtres pour les deux fères P5 et P6 de la famille MMAF3"

df_filter_fam %>% ungroup %>% mutate(`HGVSc, HGVSp` = paste0(na.replace(HGVSc, ".")," ; ", na.replace(HGVSp, "."))) %>% filter(FAM == "MMAF3", CANONICAL == "YES") %>% 
  select(SYMBOL, `HGVSc, HGVSp`, Consequence) %>%  
  distinct() %>%
  kable(format = "latex",
        booktabs = TRUE,
        longtable = TRUE,
        caption = caption) %>% 
  add_header_above(c(" ", "Variant impact" = 2))

```

```{r tabmmaf4, cache=TRUE, dependson="dfrecapmmaf"}

caption <- "Liste des variants ayant passé l'ensemble des filtres pour les deux fères P8 et P9 de la famille MMAF4"

df_filter_fam %>% ungroup %>% mutate(`HGVSc, HGVSp` = paste0(na.replace(HGVSc, ".")," ; ", na.replace(HGVSp, "."))) %>% filter(FAM == "MMAF4", CANONICAL == "YES") %>% 
  select(SYMBOL, `HGVSc, HGVSp`, Consequence) %>%  
  distinct() %>% 
  kable(format = "latex",
        booktabs = TRUE,
        longtable = TRUE,
        caption = caption) %>% 
  add_header_above(c(" ", "Variant impact" = 2))

```

\newpage

```{r tabmmaf5, cache=TRUE, dependson="dfrecapmmaf"}

caption <- "Liste des variants ayant passé l'ensemble des filtres pour le patient P10 de la famille MMAF5"

df_filter_fam %>% ungroup %>% mutate(`HGVSc, HGVSp` = paste0(na.replace(HGVSc, ".")," ; ", na.replace(HGVSp, "."))) %>% filter(FAM == "MMAF5", CANONICAL == "YES") %>% 
  select(SYMBOL, `HGVSc, HGVSp`, Consequence) %>%  
  distinct() %>% 
  kable(format = "latex",
        booktabs = TRUE,
        longtable = TRUE,
        caption = caption) %>% 
  add_header_above(c(" ", "Variant impact" = 2))

```

# Table des variants retrouvés sur le gène *PATL2*

```{r tabpatl2, dependson="dfbigmmaffilter", cache=TRUE}

caption <- "Table des variants retrouvés sur le gène \\textit{PATL2}"

df_patl2 <- bind_rows(df_ovo_filter, df_ovo_filter_het) %>%
  filter(SYMBOL == "PATL2") %>%
  filter(CANONICAL == "YES") %>%
  mutate(CALL = substr(CALL, start=1, stop=4),
         HGVSc = na.replace(HGVSc, "."),
         HGVSp = na.replace(HGVSp, "."),
         ExAC_AF = na.replace(ExAC_AF, "."),
         ESP = na.replace(ESP, "."),
         `1KG` = na.replace(`1KG`, "."),
         `ExAC ; ESP ; 1KG` = paste(ExAC_AF, ESP, `1KG`, sep = " ; "),
         `HGVSc, HGVSp` = paste(HGVSc, HGVSp, sep = " ; ")) %>%
  select(Patient = RUN, Gene = SYMBOL, Geno = CALL, `HGVSc, HGVSp`, Consequence)

   kable(df_patl2  %>%  select(-Gene) %>% arrange(desc(Geno), Patient, `HGVSc, HGVSp`),
     format = "latex",
          booktabs = TRUE,
          longtable = TRUE,
          caption = caption) %>%
  add_header_above(c(" " = 2, "Variant impact" = 2))
   
```

\newpage



# Variants retrouvés au sein de notre cohorte de patients MMAF

```{r tabdnah1, dependson="dfbigmmaffilter", cache=TRUE}

caption <- "Table des variants retrouvés sur le gène \\textit{DNAH1}"

df_dnah1 <- bind_rows(df_bigmmaf_filter, df_bigmmaf_filter_het) %>%
  filter(SYMBOL == "DNAH1") %>%
  filter(CANONICAL == "YES") %>%
  mutate(CALL = substr(CALL, start=1, stop=4),
         HGVSc = na.replace(HGVSc, "."),
         HGVSp = na.replace(HGVSp, "."),
         ExAC_AF = na.replace(ExAC_AF, "."),
         ESP = na.replace(ESP, "."),
         `1KG` = na.replace(`1KG`, "."),
         `ExAC ; ESP ; 1KG` = paste(ExAC_AF, ESP, `1KG`, sep = " ; "),
         `HGVSc, HGVSp` = paste(HGVSc, HGVSp, sep = " ; ")) %>%
  select(Patient = RUN, Gene = SYMBOL, Geno = CALL, `HGVSc, HGVSp`, Consequence, `ExAC ; ESP ; 1KG`)

  #  kable(df_dnah1  %>%  select(-Gene) %>% arrange(desc(Geno), Patient, `HGVSc, HGVSp`),
  #    format = "latex",
  #         booktabs = TRUE,
  #         longtable = TRUE,
  #         caption = caption) %>%
  # landscape() %>%
  # add_header_above(c(" " = 2, "Variant impact" = 2, "Variant frequency"))

```

\newpage

```{r tabcfap43, dependson="dfbigmmaffilter", cache=TRUE}

caption <- "Variants homozygotes retrouvés sur le gène \\textit{CFAP43}"

df_cfap43 <- df_bigmmaf_filter %>%
  filter(SYMBOL == "CFAP43") %>%
  filter(CANONICAL == "YES") %>%
  mutate(CALL = substr(CALL, start=1, stop=4),
         SIFT = na.replace(SIFT, "."),
         PolyPhen = na.replace(PolyPhen, "."),
         HGVSc = na.replace(HGVSc, "."),
         HGVSp = na.replace(HGVSp, "."),
         ExAC_AF = na.replace(ExAC_AF, "."),
         ESP = na.replace(ESP, "."),
         `1KG` = na.replace(`1KG`, "."),
         `ExAC ; ESP ; 1KG` = paste(ExAC_AF, ESP, `1KG`, sep = " ; "),
         `SIFT ; PolyPhen` = paste(SIFT, PolyPhen, sep = " ; "),
         `HGVSc, HGVSp` = paste(HGVSc, HGVSp, sep = " ; ")) %>%
  select(Patient = RUN, Geno = CALL, `HGVSc, HGVSp`, Consequence, `SIFT ; PolyPhen`, `ExAC ; ESP ; 1KG`) %>%
  mutate(`ExAC ; ESP ; 1KG` = if_else(`HGVSc, HGVSp` == "c.2658G>A ; p.Trp886Ter", "9.88e-05 ; 2e-04 ; .", `ExAC ; ESP ; 1KG`)) %>%
  arrange(`HGVSc, HGVSp`, Patient) %>% 
  distinct()

df_cfap43_het <- df_bigmmaf_filter_het %>%
  filter(SYMBOL == "CFAP43",
         CANONICAL == "YES",
         RUN == "Ghs132") %>%
  mutate(CALL = substr(CALL, start=1, stop=4),
         SIFT = na.replace(SIFT, "."),
         PolyPhen = na.replace(PolyPhen, "."),
         HGVSc = na.replace(HGVSc, "."),
         HGVSp = na.replace(HGVSp, "."),
         ExAC_AF = na.replace(ExAC_AF, "."),
         ESP = na.replace(ESP, "."),
         `1KG` = na.replace(`1KG`, "."),
         `ExAC ; ESP ; 1KG` = paste(ExAC_AF, ESP, `1KG`, sep = " ; "),
         `SIFT ; PolyPhen` = paste(SIFT, PolyPhen, sep = " ; "),
         `HGVSc, HGVSp` = paste(HGVSc, HGVSp, sep = " ; ")) %>%
  select(Patient = RUN, Geno = CALL, `HGVSc, HGVSp`, Consequence, `SIFT ; PolyPhen`, `ExAC ; ESP ; 1KG`) %>%
  
  mutate(`ExAC ; ESP ; 1KG` = if_else(`HGVSc, HGVSp` == "c.2658G>A ; p.Trp886Ter", "9.88e-05 ; 2e-04 ; .", `ExAC ; ESP ; 1KG`)) %>%
  arrange(`HGVSc, HGVSp`, Patient) %>% 
  distinct()

   kable(bind_rows(df_cfap43, df_cfap43_het) %>% arrange(desc(Geno), `HGVSc, HGVSp`, Patient),
     format = "latex",
          booktabs = TRUE,
          longtable = TRUE,
          caption = caption) %>%
     landscape() %>%
     add_header_above(c(" " = 2, "Variant impact" = 2, "Variant frequency" = 1))

```

\newpage

```{r tabcfap44, dependson="dfbigmmaffilter", cache=TRUE}

caption <- "Variants homozygotes retrouvés sur le gène \\textit{CFAP44}"

df_cfap44 <- df_bigmmaf_filter %>%
  filter(SYMBOL == "CFAP44",
         CANONICAL == "YES",
         IMPACT == "HIGH") %>% 
  mutate(CALL = substr(CALL, start=1, stop=4),
         SIFT = na.replace(SIFT, "."),
         PolyPhen = na.replace(PolyPhen, "."),
         HGVSc = na.replace(HGVSc, "."),
         HGVSp = na.replace(HGVSp, "."),
         ExAC_AF = na.replace(ExAC_AF, "."),
         ESP = na.replace(ESP, "."),
         `1KG` = na.replace(`1KG`, "."),
         `ExAC ; ESP ; 1KG` = paste(ExAC_AF, ESP, `1KG`, sep = " ; "),
         `HGVSc, HGVSp` = paste(HGVSc, HGVSp, sep = " ; "),
         `SIFT ; PolyPhen` = paste(SIFT, PolyPhen, sep = " ; ")) %>%
  select(Patient = RUN, `HGVSc, HGVSp`, Consequence, `ExAC ; ESP ; 1KG`) %>%
  arrange(`HGVSc, HGVSp`, Patient) 

kable(df_cfap44,
      format = "latex",
      booktabs = TRUE,
      longtable = TRUE,
      caption = caption) %>%
  # landscape() %>%
  add_header_above(c(" " = 1, "Variant impact" = 2, "Variant frequency" = 1))

```


# Article annexe 2 {#dpy2015}  

**Dpy19l2-deficient globozoospermic sperm display altered genome packaging and DNA damage that compromises the initiation of embryo development**


Yassine S, Escoffier J, Martinez G, Coutton C, **Karaouzène T**, Zouari R, Ravanat JL, Metzler-Guillemain C, Chang Lee H, Fissore R, Hennebicq S, Ray PF, Arnoult C  

Mol Hum Reprod, Février 2015

\newpage

\includepdf[pages=-]{bib/DPY19L2_2015}

\newpage


