<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>THÈSE</title>
  <meta name="description" content="THÈSE">
  <meta name="generator" content="bookdown 0.3.9 and GitBook 2.6.7">

  <meta property="og:title" content="THÈSE" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="THÈSE" />
  
  
  

<meta name="author" content="Thomas Karaouzene">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="B-the-second-appendix-for-fun.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="remerciements.html"><a href="remerciements.html"><i class="fa fa-check"></i>Remerciements</a></li>
<li class="chapter" data-level="" data-path="resume.html"><a href="resume.html"><i class="fa fa-check"></i>Résumé</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-introInf.html"><a href="1-introInf.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introInf.html"><a href="1-introInf.html#la-spermatogenese"><i class="fa fa-check"></i><b>1.1</b> La spermatogénèse</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-introInf.html"><a href="1-introInf.html#rappels-sur-le-testicule"><i class="fa fa-check"></i><b>1.1.1</b> Rappels sur le testicule</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-introInf.html"><a href="1-introInf.html#la-phase-de-multiplication"><i class="fa fa-check"></i><b>1.1.2</b> La phase de multiplication</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-introInf.html"><a href="1-introInf.html#la-meiose"><i class="fa fa-check"></i><b>1.1.3</b> La méïose</a></li>
<li class="chapter" data-level="1.1.4" data-path="1-introInf.html"><a href="1-introInf.html#la-spermiogenese"><i class="fa fa-check"></i><b>1.1.4</b> La spermiogénèse</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introInf.html"><a href="1-introInf.html#structure-et-fonction-du-spermatozoide"><i class="fa fa-check"></i><b>1.2</b> Structure et fonction du spermatozoïde</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-introInf.html"><a href="1-introInf.html#anatomie-du-spermatozoide"><i class="fa fa-check"></i><b>1.2.1</b> Anatomie du spermatozoïde</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-introInf.html"><a href="1-introInf.html#fonction-du-spermatozoide"><i class="fa fa-check"></i><b>1.2.2</b> Fonction du spermatozoïde</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-introInf.html"><a href="1-introInf.html#linfertilite-masculine"><i class="fa fa-check"></i><b>1.3</b> L’infertilité masculine</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-introInf.html"><a href="1-introInf.html#les-differents-phenotypes-dinfertilite-masculine"><i class="fa fa-check"></i><b>1.3.1</b> Les différents phénotypes d’infertilité masculine</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introInf.html"><a href="1-introInf.html#la-genetique-de-linfertilite"><i class="fa fa-check"></i><b>1.3.2</b> La génétique de l’infertilité</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introInf.html"><a href="1-introInf.html#les-techniques-danalyses-genetiques"><i class="fa fa-check"></i><b>1.4</b> Les techniques d’analyses génétiques</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-introInf.html"><a href="1-introInf.html#les-puces"><i class="fa fa-check"></i><b>1.4.1</b> Les puces</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-introInf.html"><a href="1-introInf.html#le-sequencage-ngs"><i class="fa fa-check"></i><b>1.4.2</b> Le séquençage NGS</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-introInf.html"><a href="1-introInf.html#lanalyse-bioinformatique-des-donnees-de-ngs"><i class="fa fa-check"></i><b>1.5</b> L’analyse bioinformatique des données de NGS</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-introInf.html"><a href="1-introInf.html#lanalyse-des-donnees-brutes"><i class="fa fa-check"></i><b>1.5.1</b> L’analyse des données brutes</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-introInf.html"><a href="1-introInf.html#lannotation-des-variants-filtrage-et-priorisation"><i class="fa fa-check"></i><b>1.5.2</b> L’annotation des variants, filtrage et priorisation</a></li>
<li class="chapter" data-level="1.5.3" data-path="1-introInf.html"><a href="1-introInf.html#conclusion-ngs"><i class="fa fa-check"></i><b>1.5.3</b> Conclusion NGS</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-globo.html"><a href="2-globo.html"><i class="fa fa-check"></i><b>2</b> Investigation génétique et physiologique de la globozoospermie</a></li>
<li class="chapter" data-level="3" data-path="3-mutascript.html"><a href="3-mutascript.html"><i class="fa fa-check"></i><b>3</b> MutaScript</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-the-first-appendix.html"><a href="A-the-first-appendix.html"><i class="fa fa-check"></i><b>A</b> The First Appendix</a></li>
<li class="chapter" data-level="B" data-path="B-the-second-appendix-for-fun.html"><a href="B-the-second-appendix-for-fun.html"><i class="fa fa-check"></i><b>B</b> The Second Appendix, for Fun</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">THÈSE</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<!--
To remove the indentation of the first entry.
-->

<!--
To create a hanging indent and spacing between entries.  These three lines may need to be removed for styles that don't require the hanging indent.
-->
<p>  </p>
<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.
-->

<div id="refs" class="references">
<div>
<p>Adelman, M. M., &amp; Cahill, E. M. (1989). <em>Atlas of sperm morphology</em> (p. 123). ASCP Press.</p>
</div>
<div>
<p>Alkan, C., Kidd, J. M., Marques-bonet, T., Aksay, G., Hormozdiari, F., Kitzman, J. O., … Eichler, E. E. (2010). Personalized Copy-Number and Segmental Duplication Maps using Next-Generation Sequencing. <em>Nature Genetics</em>, <em>41</em>(10), 1061–1067. <a href="http://doi.org/10.1038/ng.437.Personalized" class="uri">http://doi.org/10.1038/ng.437.Personalized</a></p>
</div>
<div>
<p>Asimakopoulos, B. (2003). Is There a Place for Round and Elongated Spermatids Injection in, <em>1</em>(1), 1–6.</p>
</div>
<div>
<p>Auffray, C., Chen, Z., &amp; Hood, L. (2009). Systems medicine: the future of medical genomics and healthcare. <em>Genome Medicine</em>, <em>1</em>(1), 2. <a href="http://doi.org/10.1186/gm2" class="uri">http://doi.org/10.1186/gm2</a></p>
</div>
<div>
<p>Baes, C. F., Dolezal, M. A., Koltes, J. E., Bapst, B., Fritz-Waters, E., Jansen, S., … Gredler, B. (2014). Evaluation of variant identification methods for whole genome sequencing data in dairy cattle. <em>BMC Genomics</em>, <em>15</em>(1), 948. <a href="http://doi.org/10.1186/1471-2164-15-948" class="uri">http://doi.org/10.1186/1471-2164-15-948</a></p>
</div>
<div>
<p>Bentley, D. R. (2006). Whole-genome re-sequencing. <em>Current Opinion in Genetics and Development</em>, <em>16</em>(6), 545–552. <a href="http://doi.org/10.1016/j.gde.2006.10.009" class="uri">http://doi.org/10.1016/j.gde.2006.10.009</a></p>
</div>
<div>
<p>Cho, C., Willis, W. D., Goulding, E. H., Jung-Ha, H., Choi, Y. C., Hecht, N. B., &amp; Eddy, E. M. (2001). Haploinsufficiency of protamine-1 or -2 causes infertility in mice. <em>Nature Genetics</em>, <em>28</em>(1), 82–6. <a href="http://doi.org/10.1038/88313" class="uri">http://doi.org/10.1038/88313</a></p>
</div>
<div>
<p>Clermont, Y. (1963). The cycle of the seminiferous epithelium in man. <em>American Journal of Anatomy</em>, <em>112</em>(1), 35–51. <a href="http://doi.org/10.1002/aja.1001120103" class="uri">http://doi.org/10.1002/aja.1001120103</a></p>
</div>
<div>
<p>Clermont, Y. (1966). Renewal of spermatogonia in man. <em>American Journal of Anatomy</em>, <em>118</em>(2), 509–524. <a href="http://doi.org/10.1002/aja.1001180211" class="uri">http://doi.org/10.1002/aja.1001180211</a></p>
</div>
<div>
<p>Collins, F. S., Morgan, M., &amp; Patrinos, A. (2003). The Human Genome Project: Lessons from Large-Scale Biology. <em>Science</em>, <em>300</em>(5617), 286–290. <a href="http://doi.org/10.1126/science.1084564" class="uri">http://doi.org/10.1126/science.1084564</a></p>
</div>
<div>
<p>Eddy, E. M. (2007). The scaffold role of the fibrous sheath. <em>Society of Reproduction and Fertility Supplement</em>, <em>65</em>, 45–62. Retrieved from <a href="http://www.ncbi.nlm.nih.gov/pubmed/17644954" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/17644954</a></p>
</div>
<div>
<p>Escalier, D., Gallo, J. M., Albert, M., Meduri, G., Bermudez, D., David, G., &amp; Schrevel, J. (1991). Human acrosome biogenesis: immunodetection of proacrosin in primary spermatocytes and of its partitioning pattern during meiosis. <em>Development (Cambridge, England)</em>, <em>113</em>(3), 779–788. Retrieved from <a href="http://dev.biologists.org/content/develop/113/3/779.full.pdf" class="uri">http://dev.biologists.org/content/develop/113/3/779.full.pdf</a></p>
</div>
<div>
<p>Flicek, P., &amp; Birney, E. (2009). Sense from sequence reads: methods for alignment and assembly. <em>Nature Methods</em>, <em>6</em>(11 Suppl), S6–S12. <a href="http://doi.org/10.1038/nmeth0610-479b" class="uri">http://doi.org/10.1038/nmeth0610-479b</a></p>
</div>
<div>
<p>Gnessi, L., Fabbri, A., &amp; Spera, G. (1997). Gonadal peptides as mediators of development and functional control of the testis: An integrated system with hormones and local environment. <em>Endocrine Reviews</em>, <em>18</em>(4), 541–609. <a href="http://doi.org/10.1210/er.18.4.541" class="uri">http://doi.org/10.1210/er.18.4.541</a></p>
</div>
<div>
<p>Goodwin, S., McPherson, J. D., &amp; McCombie, W. R. (2016). Coming of age: ten years of next-generation sequencing technologies. <em>Nat Rev Genet</em>, <em>17</em>(6), 333–351. <a href="http://doi.org/10.1038/nrg.2016.49" class="uri">http://doi.org/10.1038/nrg.2016.49</a></p>
</div>
<div>
<p>Goossens, E., &amp; Tournaye, H. (2013). Adult stem cells in the human testis. <em>Seminars in Reproductive Medicine</em>, <em>31</em>(1), 39–48. <a href="http://doi.org/10.1055/s-0032-1331796" class="uri">http://doi.org/10.1055/s-0032-1331796</a></p>
</div>
<div>
<p>Guffanti, A., Iacono, M., Pelucchi, P., Kim, N., Soldà, G., Croft, L. J., … De Bellis, G. (2009). A transcriptional sketch of a primary human breast cancer by 454 deep sequencing. <em>BMC Genomics</em>, <em>10</em>(1), 163. <a href="http://doi.org/10.1186/1471-2164-10-163" class="uri">http://doi.org/10.1186/1471-2164-10-163</a></p>
</div>
<div>
<p>Guo, J., Xu, N., Li, Z., Zhang, S., Wu, J., Kim, D. H., … Ju, J. (2008). Four-color DNA sequencing with 3’-O-modified nucleotide reversible terminators and chemically cleavable fluorescent dideoxynucleotides. <em>Proceedings of the National Academy of Sciences of the United States of America</em>, <em>105</em>(27), 9145–9150. <a href="http://doi.org/10.1073/pnas.0804023105" class="uri">http://doi.org/10.1073/pnas.0804023105</a></p>
</div>
<div>
<p>Hamilton, D. W., Waites, G. M. H. (1990). <em>Cellular and Molecular Events in Spermiogenesis</em> (p. 334). Cambridge University Press. Retrieved from <a href="http://www.cambridge.org/us/academic/subjects/medicine/obstetrics-and-gynecology-reproductive-medicine/cellular-and-molecular-events-spermiogenesis" class="uri">http://www.cambridge.org/us/academic/subjects/medicine/obstetrics-and-gynecology-reproductive-medicine/cellular-and-molecular-events-spermiogenesis</a></p>
</div>
<div>
<p>Hermo, L., Pelletier, R. M., Cyr, D. G., &amp; Smith, C. E. (2010). Surfing the wave, cycle, life history, and genes/proteins expressed by testicular germ cells. Part 3: Developmental changes in spermatid flagellum and cytoplasmic droplet and interaction of sperm with the zona pellucida and egg plasma membrane. <em>Microscopy Research and Technique</em>, <em>73</em>(4), 320–363. <a href="http://doi.org/10.1002/jemt.20784" class="uri">http://doi.org/10.1002/jemt.20784</a></p>
</div>
<div>
<p>Horner, D. S., Pavesi, G., Castrignano’, T., Meo, P. D. O. de, Liuni, S., Sammeth, M., … Pesole, G. (2009). Bioinformatics approaches for genomics and post genomics applications of next-generation sequencing. <em>Briefings in Bioinformatics</em>, <em>11</em>(2), 181–197. <a href="http://doi.org/10.1093/bib/bbp046" class="uri">http://doi.org/10.1093/bib/bbp046</a></p>
</div>
<div>
<p>Hwang, S., Kim, E., Lee, I., &amp; Marcotte, E. M. (2015). Systematic comparison of variant calling pipelines using gold standard personal exome variants. <em>Scientific Reports</em>, <em>5</em>(December), 17875. <a href="http://doi.org/10.1038/srep17875" class="uri">http://doi.org/10.1038/srep17875</a></p>
</div>
<div>
<p>Inaba, K. (2003). Molecular Architecture of the Sperm Flagella: Molecules for Motility and Signaling. <em>Zoological Science</em>, <em>20</em>(9), 1043–1056. <a href="http://doi.org/10.2108/zsj.20.1043" class="uri">http://doi.org/10.2108/zsj.20.1043</a></p>
</div>
<div>
<p>JOHNSON, L., PETTY, C. S., &amp; NEAVES, W. B. (1980). A Comparative Study of Daily Sperm Production and Testicular Composition in Humans and Rats. <em>Biol Reprod</em>, <em>22</em>(5), 1233–1243. Retrieved from <a href="http://www.biolreprod.org/content/22/5/1233.short" class="uri">http://www.biolreprod.org/content/22/5/1233.short</a></p>
</div>
<div>
<p>KIERSZENBAUM, A. L. (1994). Mammalian Spermatogenesis &lt;i&gt;in Vivo&lt;/i&gt; and &lt;i&gt;in Vitro&lt;/i&gt; : A Partnership of Spermatogenic and Somatic Cell Lineages*. <em>Endocrine Reviews</em>, <em>15</em>(1), 116–134. <a href="http://doi.org/10.1210/edrv-15-1-116" class="uri">http://doi.org/10.1210/edrv-15-1-116</a></p>
</div>
<div>
<p>Kierszenbaum, A. L., &amp; Tres, L. L. (1978). RNA transcription and chromatin structure during meiotic and postmeiotic stages of spermatogenesis. <em>Federation Proceedings</em>, <em>37</em>(11), 2512–6. Retrieved from <a href="http://www.ncbi.nlm.nih.gov/pubmed/357185" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/357185</a></p>
</div>
<div>
<p>Langmead, B., &amp; Salzberg, S. L. (2012). Fast gapped-read alignment with Bowtie 2. <em>Nature Methods</em>, <em>9</em>(4), 357–359. <a href="http://doi.org/10.1038/nmeth.1923" class="uri">http://doi.org/10.1038/nmeth.1923</a></p>
</div>
<div>
<p>Langmead, B., Trapnell, C., Pop, M., &amp; Salzberg, S. (2009). Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. <em>Genome Biology</em>, <em>10</em>(3), R25. <a href="http://doi.org/10.1186/gb-2009-10-3-r25" class="uri">http://doi.org/10.1186/gb-2009-10-3-r25</a></p>
</div>
<div>
<p>Lelieveld, S. H., Spielmann, M., Mundlos, S., Veltman, J. a, &amp; Gilissen, C. (2015). Comparison of Exome and Genome Sequencing Technologies for the Complete Capture of Protein-Coding Regions. <em>Human Mutation</em>, <em>36</em>(8), 815–22. <a href="http://doi.org/10.1002/humu.22813" class="uri">http://doi.org/10.1002/humu.22813</a></p>
</div>
<div>
<p>Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., … Durbin, R. (2009). The Sequence Alignment/Map format and SAMtools. <em>Bioinformatics</em>, <em>25</em>(16), 2078–2079. <a href="http://doi.org/10.1093/bioinformatics/btp352" class="uri">http://doi.org/10.1093/bioinformatics/btp352</a></p>
</div>
<div>
<p>Mardis, E. R. (2008). The impact of next-generation sequencing technology on genetics. <em>Trends in Genetics</em>, <em>24</em>(3), 133–141. <a href="http://doi.org/10.1016/j.tig.2007.12.007" class="uri">http://doi.org/10.1016/j.tig.2007.12.007</a></p>
</div>
<div>
<p>McCarthy, D. J., Humburg, P., Kanapin, A., Rivas, M. a, Gaulton, K., Cazier, J.-B., &amp; Donnelly, P. (2014). Choice of transcripts and software has a large effect on variant annotation. <em>Genome Medicine</em>, <em>6</em>(3), 26. <a href="http://doi.org/10.1186/gm543" class="uri">http://doi.org/10.1186/gm543</a></p>
</div>
<div>
<p>McKenna, A., Hanna, M., Banks, E., Sivachenko, A., Cibulskis, K., Kernytsky, A., … DePristo, M. A. (2010). The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. <em>Genome Research</em>, <em>20</em>(9), 1297–303. <a href="http://doi.org/10.1101/gr.107524.110" class="uri">http://doi.org/10.1101/gr.107524.110</a></p>
</div>
<div>
<p>McPherson, J. D. (2009). Next-generation gap. <em>Nature Methods</em>, <em>6</em>(11s), S2–S5. <a href="http://doi.org/10.1038/nmeth.f.268" class="uri">http://doi.org/10.1038/nmeth.f.268</a></p>
</div>
<div>
<p>Medvedev, P., Stanciu, M., &amp; Brudno, M. (2009). Computational methods for discovering structural variation with next-generation sequencing. <em>Nature Methods</em>, <em>6</em>(11s), S13–S20. <a href="http://doi.org/10.1038/nmeth.1374" class="uri">http://doi.org/10.1038/nmeth.1374</a></p>
</div>
<div>
<p>Meienberg, J., Bruggmann, R., Oexle, K., &amp; Matyas, G. (2016). Clinical sequencing: is WGS the better WES? <em>Human Genetics</em>, <em>135</em>(3), 359–362. <a href="http://doi.org/10.1007/s00439-015-1631-9" class="uri">http://doi.org/10.1007/s00439-015-1631-9</a></p>
</div>
<div>
<p>Metzker, M. L. (2010). Sequencing technologies - the next generation. <em>Nature Reviews. Genetics</em>, <em>11</em>(1), 31–46. <a href="http://doi.org/10.1038/nrg2626" class="uri">http://doi.org/10.1038/nrg2626</a></p>
</div>
<div>
<p>Ng, S. B., Turner, E. H., Robertson, P. D., Flygare, S. D., Abigail, W., Lee, C., … Shendure, J. (2010). Targeted Capture and Massicely Parallel Sequencing of twelve human exomes. <em>Nature</em>, <em>461</em>(7261), 272–276. <a href="http://doi.org/10.1038/nature08250.Targeted" class="uri">http://doi.org/10.1038/nature08250.Targeted</a></p>
</div>
<div>
<p>Nielsen, R., Paul, J. S., Albrechtsen, A., &amp; Song, Y. S. (2011). Genotype and SNP calling from next-generation sequencing data. <em>Nature Reviews. Genetics</em>, <em>12</em>(6), 443–51. <a href="http://doi.org/10.1038/nrg2986" class="uri">http://doi.org/10.1038/nrg2986</a></p>
</div>
<div>
<p>Ogura, a, Matsuda, J., &amp; Yanagimachi, R. (1994). Birth of normal young after electrofusion of mouse oocytes with round spermatids. <em>Proceedings of the National Academy of Sciences of the United States of America</em>, <em>91</em>(16), 7460–7462. <a href="http://doi.org/10.1073/pnas.91.16.7460" class="uri">http://doi.org/10.1073/pnas.91.16.7460</a></p>
</div>
<div>
<p>Ogura, A., Matsuda, J., Asano, T., Suzuki, O., &amp; Yanagimachi, R. (1996). Mouse oocytes injected with cryopreserved round spermatids can develop into normal offspring. <em>Journal of Assisted Reproduction and Genetics</em>, <em>13</em>(5), 431–434. <a href="http://doi.org/10.1007/BF02066177" class="uri">http://doi.org/10.1007/BF02066177</a></p>
</div>
<div>
<p>O’Rawe, J., Jiang, T., Sun, G., Wu, Y., Wang, W., Hu, J., … Lyon, G. J. (2013). Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing. <em>Genome Medicine</em>, <em>5</em>(3), 28. <a href="http://doi.org/10.1186/gm432" class="uri">http://doi.org/10.1186/gm432</a></p>
</div>
<div>
<p>Papic, Z., Katona, G., &amp; Skrabalo, Z. (1988). The cytologic identification and quantification of testicular cell subtypes. Reproducibility and relation to histologic findings in the diagnosis of male infertility. <em>Acta Cytologica</em>, <em>32</em>(5), 697–706. Retrieved from <a href="http://www.ncbi.nlm.nih.gov/pubmed/3421018" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/3421018</a></p>
</div>
<div>
<p>Pedersen, H., &amp; Rebbe, H. (1974). Fine structure of round-headed human spermatozoa. <em>Journal of Reproduction and Fertility</em>, <em>37</em>(1), 51–4. <a href="http://doi.org/10.1530/JRF.0.0370051" class="uri">http://doi.org/10.1530/JRF.0.0370051</a></p>
</div>
<div>
<p>Pruitt, K. D., Harrow, J., Harte, R. A., Wallin, C., Diekhans, M., Maglott, D. R., … Lipman, D. (2009). The consensus coding sequence (CCDS) project: Identifying a common protein-coding gene set for the human and mouse genomes. <em>Genome Research</em>, <em>19</em>(7), 1316–1323. <a href="http://doi.org/10.1101/gr.080531.108" class="uri">http://doi.org/10.1101/gr.080531.108</a></p>
</div>
<div>
<p>Qin, J., Li, R., Raes, J., Arumugam, M., Burgdorf, S., Manichanh, C., … Yang, H. (2010). A human gut microbial gene catalog established by metagenomic sequencing. <em>Nature</em>, <em>464</em>(7285), 59–65. <a href="http://doi.org/10.1038/nature08821.A" class="uri">http://doi.org/10.1038/nature08821.A</a></p>
</div>
<div>
<p>Rosenfeld, J. A., Mason, C. E., Smith, T. M., Wallin, C., &amp; Diekhans, M. (2012). Limitations of the Human Reference Genome for Personalized Genomics. <em>PLoS ONE</em>, <em>7</em>(7), e40294. <a href="http://doi.org/10.1371/journal.pone.0040294" class="uri">http://doi.org/10.1371/journal.pone.0040294</a></p>
</div>
<div>
<p>Sasagawa, I., &amp; Yanagimachi, R. (1997). Spermatids from mice after cryptorchid and reversal operations can initiate normal embryo development. <em>Journal of Andrology</em>, <em>18</em>(2), 203–209. Retrieved from <a href="http://www.ncbi.nlm.nih.gov/pubmed/9154515" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/9154515</a></p>
</div>
<div>
<p>Schatz, M. C., &amp; Langmead, B. (2013). The DNA Data Deluge: Fast, efficient genome sequencing machines are spewing out more data than geneticists can analyze. <em>IEEE Spectrum</em>, <em>50</em>(7), 26–33. <a href="http://doi.org/10.1109/MSPEC.2013.6545119" class="uri">http://doi.org/10.1109/MSPEC.2013.6545119</a></p>
</div>
<div>
<p>Schenck, U., &amp; Schill, W. B. (n.d.). Cytology of the human seminiferous epithelium. <em>Acta Cytologica</em>, <em>32</em>(5), 689–96. Retrieved from <a href="http://www.ncbi.nlm.nih.gov/pubmed/3421017" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/3421017</a></p>
</div>
<div>
<p>Sims, D., Sudbery, I., Ilott, N. E., Heger, A., &amp; Ponting, C. P. (2014). Sequencing depth and coverage: key considerations in genomic analyses. <em>Nature Reviews. Genetics</em>, <em>15</em>(2), 121–32. <a href="http://doi.org/10.1038/nrg3642" class="uri">http://doi.org/10.1038/nrg3642</a></p>
</div>
<div>
<p>Singh, G. (n.d.). Ultrastructural features of round-headed human spermatozoa. <em>International Journal of Fertility</em>, <em>37</em>(2), 99–102. Retrieved from <a href="http://www.ncbi.nlm.nih.gov/pubmed/1349598" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/1349598</a></p>
</div>
<div>
<p>Sultan, M., Schulz, M. H., Richard, H., Magen, A., Klingenhoff, A., Scherf, M., … Yaspo, M.-L. (2008). A Global View of Gene Activity and Alternative Splicing by Deep Sequencing of the Human Transcriptome. <em>Science</em>, <em>321</em>(5891), 956–960. <a href="http://doi.org/10.1126/science.1160342" class="uri">http://doi.org/10.1126/science.1160342</a></p>
</div>
<div>
<p>Tanaka, A., Nagayoshi, M., Takemoto, Y., Tanaka, I., Kusunoki, H., Watanabe, S., … Yanagimachi, R. (2015). Fourteen babies born after round spermatid injection into human oocytes. <em>Proceedings of the National Academy of Sciences</em>, <em>112</em>(March 2014), 201517466. <a href="http://doi.org/10.1073/pnas.1517466112" class="uri">http://doi.org/10.1073/pnas.1517466112</a></p>
</div>
<div>
<p>Taylor, K. H., Kramer, R. S., Davis, J. W., Guo, J., Duff, D. J., Xu, D., … Shi, H. (2007). Ultradeep Bisulfite Sequencing Analysis of DNA Methylation Patterns in Multiple Gene Promoters by 454 Sequencing. <em>Cancer Research</em>, <em>67</em>(18), 8511–8518. <a href="http://doi.org/10.1158/0008-5472.CAN-07-1016" class="uri">http://doi.org/10.1158/0008-5472.CAN-07-1016</a></p>
</div>
<div>
<p>Tomkinson, A. E., Vijayakumar, S., Pascal, J. M., &amp; Ellenberger, T. (2006). DNA Ligases:  Structure, Reaction Mechanism, and Function. <em>Chemical Reviews</em>, <em>106</em>(2), 687–699. <a href="http://doi.org/10.1021/cr040498d" class="uri">http://doi.org/10.1021/cr040498d</a></p>
</div>
<div>
<p>Treangen, T. J., &amp; Salzberg, S. L. (2013). Repetitive DNA and next-generation sequencing: computational challenges and solutions. <em>Nat Rev Genet.</em>, <em>13</em>(1), 36–46. <a href="http://doi.org/10.1038/nrg3117.Repetitive" class="uri">http://doi.org/10.1038/nrg3117.Repetitive</a></p>
</div>
<div>
<p>Van Tassell, C. P., Smith, T. P. L., Matukumalli, L. K., Taylor, J. F., Schnabel, R. D., Lawley, C. T., … Sonstegard, T. S. (2008). SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries. <em>Nature Methods</em>, <em>5</em>(3), 247–252. <a href="http://doi.org/10.1038/nmeth.1185" class="uri">http://doi.org/10.1038/nmeth.1185</a></p>
</div>
<div>
<p>Ward, W. S. (1994). The structure of the sleeping genome: implications of sperm DNA organization for somatic cells. <em>Journal of Cellular Biochemistry</em>, <em>55</em>(1), 77–82. <a href="http://doi.org/10.1002/jcb.240550109" class="uri">http://doi.org/10.1002/jcb.240550109</a></p>
</div>
<div>
<p>Wold, B., &amp; Myers, R. M. (2007). Sequence census methods for functional genomics. <em>Nature Methods</em>, <em>5</em>(1), 19–21. <a href="http://doi.org/10.1038/nmeth1157" class="uri">http://doi.org/10.1038/nmeth1157</a></p>
</div>
<div>
<p>World Health Organization. (1992). <em>WHO laboratory manual for the examination of human semen and sperm-cervical mucus interaction.</em> (3th ed, p. 128). Cambridge University Press.</p>
</div>
<div>
<p>Yang, M. Q., Athey, B. D., Arabnia, H. R., Sung, A. H., Liu, Q., Yang, J. Y., … Deng, Y. (2009). High-throughput next-generation sequencing technologies foster new cutting-edge computing techniques in bioinformatics. <em>BMC Genomics</em>, <em>10 Suppl 1</em>, I1. <a href="http://doi.org/10.1186/1471-2164-10-S1-I1" class="uri">http://doi.org/10.1186/1471-2164-10-S1-I1</a></p>
</div>
<div>
<p>Zhao, S., &amp; Zhang, B. (2015). A comprehensive evaluation of ensembl, RefSeq, and UCSC annotations in the context of RNA-seq read mapping and gene quantification. <em>BMC Genomics</em>, <em>16</em>(1), 97. <a href="http://doi.org/10.1186/s12864-015-1308-8" class="uri">http://doi.org/10.1186/s12864-015-1308-8</a></p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="B-the-second-appendix-for-fun.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
